src/containers/BooksForm.js

```diff
@@ -1,14 +1,40 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { createBook } from '../actions';
+
 export default function BooksForm() {
+  const dispatch = useDispatch();
+
+  const initialState = {
+    title: '',
+    categoryName: 'Action',
+  };
+  const [form, updateInput] = useState(initialState);
+
```

Comment:
[OPTIONAL]

- [ ] Please let's adopt the ES6 arrow function syntax in all the components.